advanced/list-of-log-names.md

```diff
@@ -4,43 +4,43 @@ description: On this page all the available logs are listed
+Logs, categorized by what they do are listed here, and are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names (which can be used in place of individual log names, see [_**configuring channels**_](../management-commands/config.md)) are _Capitalized_. Each one of them is separated by categories in the lists below. If you want to log everything, use an asterisk to include all things that can be logged, including ones that will be introduced in future updates.
 
 {% hint style="warning" %}
-Some logs require the bot to have the "View Audit Logs" permission enabled.
+Some logs require the bot to have the "View Audit Log" permission enabled.
 {% endhint %}
 
 ## Moderation
 
-This category is used for actions by moderators or ones that moderators could use. Most of them are fired when a command is run unless they start with AUDIT_. In that case, actions that are done with or without using the bot are logged.
+This category is used for actions by moderators or ones that moderators could use. Most of them are fired when a command is run unless they start with AUDIT_, which uses the values in the server's audit log. In that case, actions that are done with or without using the bot are logged.
 
 | Name              | Emitted when     |
 | ----------------- | ---------------- |
-| BAN               | Someone gets banned |
-| INFRACTION_DELETE | Someone removes an infraction|
-| KICK              | Someone gets kicked |
-| MUTE              | Someone gets muted |
-| PAUSE_ROLE_ADD    | An expiring role gets added  |
-| PAUSE_ROLE_EXTEND | A role expiration date gets extended |
-| PAUSE_ROLE_REMOVE | A role gets temporarily removed |
+| BAN               | A user gets banned |
+| INFRACTION_DELETE | A moderator removes an infraction|
+| KICK              | A user gets kicked |
+| MUTE              | A user gets muted |
+| PAUSE_ROLE_ADD    | A role gets temporarily removed |
+| PAUSE_ROLE_EXTEND | A role's length of time to be removed gets adjusted |
+| PAUSE_ROLE_REMOVE | A role gets added back |
 | REASON_UPDATE     | A reason for an infraction gets updated |
-| ROLE_ADD          | Someone adds a role |
-| ROLE_REMOVE       | Someone removes a role|
-| SELFROLE_ADD      | Someone adds a user-assignable role |
-| SELFROLE_REMOVE   | Someone removes a user-assignable role  |
-| SOFTBAN           | Someone gets soft-banned |
-| TEMPBAN_EXTEND    | Someone's temporary ban gets extended |
-| TEMPBAN_REMOVE    | Someone's temporary ban gets removed |
-| TEMPMUTE_EXTEND   | Someone's temporary mute gets extended |
-| TEMPMUTE_REMOVE   | Someone's temporary ban gets removed |
+| ROLE_ADD          | A role gets added |
+| ROLE_REMOVE       | A role gets removed|
+| SELFROLE_ADD      | A user adds a self-assignable role (see [Selfrole](../role-management-commands/selfrole.md)) |
```

Comment:
I prefer the Wikipedia style of linking directly in the words without adding content to make them clickable.

E.g. `A user adds a [self-assignable role](../role-management-commands/selfrole.md)`

And conclude the sentence with "to themselves".